frontend/src/lib/utils.js

```diff
@@ -241,6 +241,13 @@ export const deletePersonData = (person, callback) => {
         })
 }
 
+export const savePersonData = (person) => {
+    window.confirm("save person's data") &&
+        api.update('api/person/' + person.id).then(() => {
```

Comment:
@cr33dx You will get a 200 on the PATCH request even if you do not send any data. Here, `update` takes `data` as a second parameter